bot/cogs/bot.py

```diff
@@ -63,11 +73,7 @@ def __init__(self, bot: AutoShardedBot):
         embed.add_field(name="Total Users", value=str(len(self.bot.get_guild(PYTHON_GUILD).members)))
         embed.add_field(name="Git SHA", value=str(sha)[:7])
 
-        embed.set_author(
-            name="Python Bot",
-            url="https://github.com/discord-python/bot",
-            icon_url=BOT_AVATAR_URL
-        )
+        embed.set_author(name="Python Bot", url="https://github.com/discord-python/bot", icon_url=BOT_AVATAR_URL)
```

Comment:
I still prefer the old one here. whenever we list things like this, it's nice to multiline it.